tasks/__init__.py

```diff
@@ -443,70 +432,69 @@ def test(ctx, all=False, syntax=False):
         test_admin(ctx)
         karma(ctx)
 
-
 @task
-def test_js(ctx):
-    jshint(ctx)
-    karma(ctx)
+def travis_setup(ctx):
+    ctx.run('npm install -g bower', echo=True)
+
+    with open('package.json', 'r') as fobj:
+        package_json = json.load(fobj)
+        ctx.run('npm install @centerforopenscience/list-of-licenses@{}'.format(package_json['dependencies']['@centerforopenscience/list-of-licenses']), echo=True)
+
+    with open('bower.json', 'r') as fobj:
+        bower_json = json.load(fobj)
+        ctx.run('bower install {}'.format(bower_json['dependencies']['styles']), echo=True)
 
 @task
 def test_travis_addons(ctx, numprocesses=None, coverage=False):
     """
     Run half of the tests to help travis go faster. Lints and Flakes happen everywhere to keep from wasting test time.
     """
+    travis_setup(ctx)
     flake(ctx)
-    jshint(ctx)
     test_addons(ctx, numprocesses=numprocesses, coverage=coverage)
 
-
 @task
-def test_travis_else(ctx, numprocesses=None, coverage=False):
+def test_travis_website(ctx, numprocesses=None, coverage=False):
     """
     Run other half of the tests to help travis go faster. Lints and Flakes happen everywhere to keep from
     wasting test time.
     """
-    flake(ctx)
-    jshint(ctx)
-    test_else(ctx, numprocesses=numprocesses, coverage=coverage)
+    # flake(ctx)
```

Comment:
This is certainly an improvement. I think we could go one step further and run flake8 in `before_install` as the first step. That way, we get a more immediate failure.

Doesn't necessarily have to happen in this PR though.